octopoes/octopoes/connector/octopoes.py

```diff
@@ -38,8 +39,8 @@ class OctopoesAPIConnector:
     def __init__(self, base_uri: str, client: str):
         self.base_uri = base_uri
         self.client = client
-
         self.session = httpx.Client(base_url=base_uri, timeout=30, event_hooks={"response": [self._verify_response]})
+        self.logger = structlog.get_logger("octopoes-connector", client=client)
```

Comment:
The client object doesn't have a string representation, so this doesn't seem useful? The organisation code can be added in the OrganisationView. We can also use a module level logger here in that case.

Reply:
The `client` here is a string representation (of a node).

> The organisation code can be added in the OrganisationView

This would work within the context of Rocky, but this client is also used elsewhere, e.g. by the Boefje workers

Reply:
Ok, but then we should call it `organization_code` and not use different keys for the same thing, because that makes it difficult/impossible to filter/search log messages.

```suggestion
        self.logger = structlog.get_logger("octopoes-connector", organisation_code=client)
```

Reply:
You're absolutely right, will commit the suggestion